core/conf/conf.go

```diff
@@ -0,0 +1,125 @@
+// pakcage conf provide config helpers for ~/.zcn/config.yaml, ～/.zcn/network.yaml and ~/.zcn/wallet.json
+
+package conf
+
+import (
+	"errors"
+	"net/url"
+	"os"
+	"path"
+	"strings"
+
+	thrown "github.com/0chain/errors"
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+)
+
+var (
+	// ErrMssingConfig config file is missing
+	ErrMssingConfig = errors.New("[conf]missing config file")
+	// ErrInvalidValue invalid value in config
+	ErrInvalidValue = errors.New("[conf]invalid value")
+	// ErrBadFormat fail to parse config via spf13/viper
+	ErrBadFormat = errors.New("[conf]bad format")
+)
+
+// LoadDefault load and parse config from ~/.zcn/config.yaml
+func LoadDefault() (Config, error) {
+	return Load("config.yaml")
```

Comment:
I think it's better and safety to pass full path to config, instead of just filename